gluon/tests/test_scheduler.py

```diff
@@ -309,12 +309,12 @@ def testRepeats_and_Expired_and_Prio(self):
         self.db.commit()
         self.writefunction(r"""
 def demo1(*args,**vars):
-    print 'you passed args=%s and vars=%s' % (args, vars)
+    print('you passed args=%s and vars=%s') % (args, vars)
```

Comment:
Same thing here : 

I find this cleaner : 
print('you passed args=%s and vars=%s' % (args, vars))

Or even : 
print('you passed args={} and vars={}'.format(args, vars))